website/blog/2024-03-26-discord-setup/index.mdx

```diff
@@ -0,0 +1,170 @@
+
+## 10 - Let’s Talk Intermediate and Advanced features 🤓
+
+All of that seemed pretty straightforward, right? In case you are looking to get fancy, let’s explore a couple of intermediate and advanced features. 
+
+>💡 Discord has a subscription plan which uses `Nitro` as currency, Nitro can be bought with fiat currency. By buying boosts you can unlock many features and add-ons, mainly cosmetic and image quality-related, but feel free to check them out.
+
+![Discord boosts](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/em78bznbkl4e0q5z2niw.png)
+
+### **Intermediate**
+
+It might be the case that over time your server grows and so do the number of channels. Since a large number of available channels can be disorientating for new members you can allow members to opt-in through additional onboarding questions. 
+
+> 💡 Note that even if you don’t answer the questions during onboarding you can always click on `Channels and Roles` in the top left-hand corner of the server to answer any onboarding questions to gain access to any locked channels.
+
+To get started, navigate to `Server Settings> Community Onboarding > Default channels`, and only select the channels you want all members to see.
+
+
+![Intermediate onboarding](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/2e85wi3frunfqlkgqihl.png)
+
+Then edit the onboarding questions and unlock channels depending on the answer by adding the channel to part 2 `Assign channels and/or roles` 
+
+
+![Unlock channels](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/35ybhzbplzubgyy4x9ds.png)
+
+### **Advanced**
+
+Similarly, if you have a growing server with multiple announcement channels that you want to give granular access to. For example, if a user with the `@contributor` role only wants `Glasskubee updates` and not `general Kubernetes updates` you can gain this level of granularity by switching to `Advanced mode` in the Onboarding section of the `Server settings` page. 
+
+![Slowmode](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/g67x4vqcsqxvtsrwiewq.png)
+
+You can also be granular about the amount of time is needed for a new member to send a message in a particular channel by editing the `Slowmode` setting, under channel settings.
+
+## Let’s Wrap It Up
+
+A community platform is a place that should be conducive to creativity, comradery, and collaboration. Of course, some rules and guidelines should be in place to keep the place welcoming and inclusive to all, but all in all the key takeaway is to not overthink or over-engineer your server. 
+Decide on some key roles, and key channels and resolve people's issues in a centralized way with answers to questions easy to find and hopefully don’t always depend on your input. 
+
+A community is made up of individual connections to try to focus more on initiatives that will lead to open conversation and always try to acknowledge key community members when they deserve it.
```

Comment:
I would recommend a call to action section similar to the release blog post (but slightly adapted):

## Get involved

The easiest way to get involved is to tackle one of our [open issues](https://github.com/glasskube/glasskube/issues).
You are also welcome to join our [Discord](https://discord.gg/SxH6KUCGH7).

> **If you are a cloud native developer, please [submit your package](https://github.com/glasskube/glasskube/discussions/90)**.

As Glasskube is still in its very early days, your feedback is highly appreciated.
Let us know what you think, we would love to hear from you or support us by leaving a star on GitHub:

[⭐ `glasskube/glasskube` ⭐](https://github.com/glasskube/glasskube/)